tools/compile-keymap.c

```diff
@@ -244,8 +244,10 @@ print_keymap(struct xkb_context *ctx, const struct xkb_rule_names *rmlvo)
     if (keymap == NULL)
         return false;
 
-    printf("%s\n", xkb_keymap_get_as_string(keymap,
-                                            XKB_KEYMAP_FORMAT_TEXT_V1));
+    char *buf = xkb_keymap_get_as_string(keymap, XKB_KEYMAP_FORMAT_TEXT_V1);
```

Comment:
@bluetech are we happy to go C99? Because just these late declarations are going to make the code better in a lot of places but let's make this official.

Reply:
No problem with using C99 late declarations from me.